.babelrc

```diff
@@ -1,7 +1,8 @@
 {
   "presets": [
-    "es2015",
-    "es2017"
+    ["env", {
```

Comment:
👍 for using env preset but i think we need to explicit state version what we want to support. In this case I think it will use your node version as a baseline and we definately dont want it on prod (it impacts all dependent projects). I think we should target something around node 6.